src/js/components/Calendar/__tests__/Calendar-test.js

```diff
@@ -113,10 +116,37 @@ describe('Calendar', () => {
               </Button>
             </Box>
           )}
+          animate={false}
         />
       </Grommet>,
     );
     const tree = component.toJSON();
     expect(tree).toMatchSnapshot();
   });
+
+  test('select date', () => {
+    const onSelect = jest.fn();
+    const { getByText, container } = render(
+      <Grommet>
+        <Calendar date={DATE} onSelect={onSelect} animate={false} />
+      </Grommet>,
+    );
+    expect(container.firstChild).toMatchSnapshot();
+    fireEvent.click(getByText('17'));
+    expect(onSelect).toBeCalledWith(expect.stringMatching(/^2018-01-17T/));
+    expect(container.firstChild).toMatchSnapshot();
+  });
+
+  test('select dates', () => {
```

Comment:
This test looks the same as the previous test except it is missing the `animate`, wasn't sure if that was on purpose.
I would think that this test should simulate more than one date selection since the test name has `dates` (plural).

Reply:
All should have `animate={false}`.

Calendar `onSelect` always provides the date the user clicked on. It's up to the caller to decide what to do with it. But, internally it is different whether you start with a single date or multiple dates.

Reply:
got it, thanks for clarifying.

Reply:
...added `animate={false}`.